platform/features/conductor/core/src/ui/TimeConductorController.js

```diff
@@ -167,23 +166,26 @@ define(
          * @param oldOption
          */
         TimeConductorController.prototype.selectMenuOption = function (newOption, oldOption) {
```

Comment:
Parameter oldOption isn't used, not sure if left in for legacy reasons but thought I'd point this out.